### PR TITLE
Home Task 2

### PR DESCRIPTION
**Pull request by MatthewZholud**

### CODE REVIEW

File: HomeTask2/Task2_2.go [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+		if len(a) > len(word) {
+			word = a
+		}
+	}
+	return word
+}
+
+func main() {
+	sl := []string{"one", "two", "three"}
+	//sl := []string{"one", "two"}
```
Comment (ikamornikau) [inline HomeTask2/Task2_2.go]: Don't keep redundant comments, please.

**Review by ikamornikau — APPROVED** [review]

File: HomeTask2/Task2_1.go [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package main
+
+import "fmt"
+
+func average(arr [6]int) float64 {
+	var sum float64
+	for _, a := range arr {
+		sum += float64(a)
+	}
+	return sum / float64(len(arr))
```
Comment (wshaman) [inline HomeTask2/Task2_1.go:10]: :+1:

File: HomeTask2/Task2_2.go [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+	word := sl[0]
+	for _, a := range sl {
+		if len(a) > len(word) {
+			word = a
+		}
+	}
+	return word
+}
+
+func main() {
```
Comment (wshaman) [inline HomeTask2/Task2_2.go:15]: Usually it's not the best idea to have multiply `main` funcs in one package. In fact, it'll be a conflict.
Though, it'll work while running `go run filename.go`, but not advised.

File: HomeTask2/Task2_2.go [hunk 4]
```diff
@@ -0,0 +1,18 @@
+package main
+
+import "fmt"
+
+func max(sl []string) string {
+	word := sl[0]
```
Comment (wshaman) [inline HomeTask2/Task2_2.go:6]: `sl` can be empty. What this code will do?

File: HomeTask2/Task2_3.go [hunk 5]
```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"fmt"
+)
+
+func reverse(slc []int64) []int64 {
+	slc1 := make([]int64, cap(slc))
+	copy(slc1, slc)
```
Comment (wshaman) [inline HomeTask2/Task2_3.go:9]: :+1:

**Review by wshaman — APPROVED** [review]

**Pull request merged**